client/state/data-layer/wpcom/sites/plugins/new/index.js

```diff
@@ -21,6 +28,38 @@ export const uploadPlugin = ( { dispatch }, action ) => {
 	}, action ) );
 };
 
+const showSuccessNotice = ( dispatch, { name } ) => {
+	dispatch( successNotice(
+		translate( 'Successfully uploaded plugin %(name)s.', {
+			args: { name }
+		} ),
+		{ duration: 5000 }
+	) );
+};
+
+const showErrorNotice = ( dispatch, error ) => {
+	const knownErrors = {
+		exists: translate( 'Upload problem: Plugin already installed on site.' ),
+		'too large': translate( 'Upload problem: Plugin zip must be smaller than 10MB.' ),
+		incompatible: translate( 'Upload problem: Incompatible plugin.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Upload problem: Incompatible theme.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).